script.video.bgtime_tv/addon.xml

```diff
@@ -29,6 +28,6 @@
 		</disclaimer>
 		<website>https://bgtime.tv/</website>
 		<email>support@mytv.bg</email>
```

Comment:
is this an official addon? If not I doubt mytv.bg will be happy to receive emails regarding the plugin